### DISCUSSION TITLE
High CPU usage when watching a movie

### DISCUSSION THREAD

**Discussion by Sevinfolds** [jellyfin/jellyfin#12828] [Q&A]
Today, for the first time, I installed Jellyfin 10.9.11 on ubuntu 22.04 and configured the apache2 reverse proxy for external use via https. Everything works well, but after deciding to see how much resources watching a movie in a browser consumes (mkv from 8 to 15Gb), I was surprised, my i5 13500 was loaded by 40%, running another movie in parallel, the download turned out to be about 80%. Tell me, is this how it should be, or did you not configure something?

**Comment by cvium** [comment]
I don't understand that last part, but if you are watching something in the browser, Jellyfin is likely transcoding the media. The high CPU usage is to be expected in that case.

**Reply by Sevinfolds** [reply]
Yes, that's right, I looked through the browser. If I understood correctly, then this transcoding cannot be disabled, or do I need a video card? A simple example, I run the same movies in my nextcloud cloud (also in the browser), there is practically no load, less than 5%.

**Reply by TheMelmacian** [reply]
If you need a video card for hardware transcoding depends on your CPU model. If it has an integrated GPU with the necessary codec support there is no need for a dedicated GPU. You need to activate hardware transcoding in the settings.
https://jellyfin.org/docs/general/administration/hardware-acceleration/
Only activate codecs that are supported by the GPU.
